### PR TITLE
feat: add icon, meta, and og for seo

### PR DESCRIPTION
**Pull request by lewxdev**
https://github.com/user-attachments/assets/4c56533b-bc4a-420f-9d63-6530836b89fa

edit: tough shit https://github.com/superfly/fly-pr-review-apps/issues/4

### CODE REVIEW

File: .github/workflows/fly-review.yml [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
       name: review
-      # The script in the `deploy` sets the URL output for each review app.
       url: ${{ steps.deploy.outputs.url }}
 
     steps:
       - uses: actions/checkout@v4
-
       - id: deploy
-        uses: superfly/fly-pr-review-apps@1.2.1
+        uses: JargeZ/fly-pr-review-apps@4d6566e85d6b9320a781fb4fe42f1f8c31dc5c8e
```
Comment (drewradcliff) [inline .github/workflows/fly-review.yml]: Can you fork this yourself?

**Comment by lewxdev** [comment]
@drewradcliff it's ready

File: .github/workflows/fly-review.yml (hunk 1 above)
Comment (lewxdev) [reply .github/workflows/fly-review.yml]: I guess I can, but do you really want to put me through that? 🥲

**Review by drewradcliff — APPROVED** [review]
🚀

**Pull request merged**